source/agent/addons/quic/QuicTransportConnection.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef QUIC_QUICTRANSPORTCONNECTION_H_
+#define QUIC_QUICTRANSPORTCONNECTION_H_
+
+#include <memory>
+#include <mutex>
+#include <unordered_map>
+#include <vector>
+
+#include <logger.h>
+#include <nan.h>
+
+#include "QuicTransportStream.h"
+#include "owt/quic/quic_transport_session_interface.h"
+
+class QuicTransportConnection : public Nan::ObjectWrap, public owt::quic::QuicTransportSessionInterface::Visitor, QuicTransportStream::Visitor {
```

Comment:
What's the point of inherriting from QuicTransportStream::Visitor while not implementing it?  And more, your visitor callback does not map stream in OnEnded callback, so even if you implement the OnEnded callback, how to you identify the stream that actually fires this event?